### DISCUSSION TITLE
Multiple bounding boxes in snapshots in case of multiple objects ?

### DISCUSSION THREAD

**Discussion by bonjour81** [blakeblackshear/frigate#15409] [Ask A Question] [answered]
Hello,

I was wondering if it is possible or planned to have multiple bounding boxes in case more than one object if detected.
For example if 2 peoples are detected, one for each ?

if already exists, how to enable this?

**Comment by NickM-27** [comment, marked as answer]
No, a snapshot exists for each object separately. You can look at the debug view to see all objects that are detected
